src/Paket.Core/Dependencies/PackageResolver.fs

```diff
@@ -681,7 +726,7 @@ let Resolve (getVersionsF, getPackageDetailsF, groupName:GroupName, globalStrate
                         let flags = 
                             StepFlags(flags.Ready,flags.UseUnlisted,flags.HasUnlisted,true,flags.FirstTrial,flags.UnlistedSearch)
                        
-                        step (Inner((continueConflict,lastStep,lastRequirement),priorConflictSteps)) stackpack lastCompatibleVersions  lastFlags 
+                        step (Inner((continueConflict,lastStep,lastRequirement),priorConflictSteps)) stackpack lastCompatibleVersions  flags 
```

Comment:
now that I actually have the code in front of me in VS, that's the force break flag, so that should just bail us out of resolution faster and it probably shouldn't make much of a difference what the other flags are (don't quote me on that ;P)